pyvo/dal/__init__.py

```diff
@@ -19,7 +21,7 @@
 
 __all__ = [
     "imagesearch", "spectrumsearch", "linesearch", "conesearch", "tablesearch",
-    "DALService",
+    "DALService", "sia2", "imagesearch2",
```

Comment:
What is the purpose of `__all__` here? Probably to define the scope of `from pyvo.dal import *`. If that's the case, then `sia2` is required otherwise `sia2` is not included in the wildcard set. I don't know what other purpose can `__all__` serve here.

Reply:
Ah now i got it. Again, no strong opinion how to do it, it should just be one way only.
Will approve both ways.